docs/developers/how-to-build-plugins.md

```diff
@@ -2,82 +2,111 @@
 sidebar-position: 1
 ---
 
-# How to build model plugins
+# How to build plugins
 
-The IF is designed to be as composable as possible. This means you can develop your own models and use them in a model pipeline.
-To help developers write Typescript models to integrate easily into IF, we provide the `ModelPluginInterface` interface. Here's an overview of the stages you need to follow to integrate your model:
+The IF is designed to be as composable as possible. This means you can develop your own plugins and use them in a pipeline.
+To help developers write Typescript plugins to integrate easily into IF, we provide the `PluginInterface` interface. Here's an overview of the stages you need to follow to integrate your plugin:
 
-- create a Typescript file that implements the `ModelPluginInterface`
-- install the model
-- initialize and invoke the model in your manifest file
+- create a Typescript file that manifestements the `PluginInterface`
```

Comment:
@jmcook1186 Please check , seems that it should've been 'implements' instead of 'manifestements'. It appears multiple times